hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/datanode/fsdataset/impl/FsDatasetCache.java

```diff
@@ -184,6 +186,26 @@ public FsDatasetCache(FsDatasetImpl dataset) throws IOException {
     this.memCacheStats = cacheLoader.initialize(this.getDnConf());
   }
 
+  public void restoreCache(String bpid) throws IOException {
```

Comment:
Please add javadoc for this method. You can refer to that for other method. Applicable to all newly add method.